lib/static/new-ui/features/suites/selectors.ts

```diff
@@ -57,6 +57,22 @@ export const getCurrentResultImages = (state: State): ImageEntity[] => {
     return result?.imageIds.map(imageId => state.tree.images.byId[imageId]) ?? [];
 };
 
+export const getCurrentSuiteId = (params: Record<string, string | undefined>): ((state: State) => string | null) => {
```

Comment:
I dont like how "getCurrentSuiteId" returns `currentSuite.id + ' ' + params.browser`
If its "getCurrentSuiteId", it should only return "currentSuite.id"
If it returns `currentSuite.id + ' ' + params.browser`, it should not be called "getCurrentSuiteId"

Reply:
Yea, I had bad naming, change all suiteId to currentBrowserId for screenshots page, in suite page we have valid naming